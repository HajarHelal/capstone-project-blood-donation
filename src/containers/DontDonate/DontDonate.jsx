import React from 'react';
import image from './images/doctor-office-and-blood-donation-design-vector-22887400.jpg';
import { useTranslation } from 'react-i18next';

function DontDonate() {
  const { t } = useTranslation();
  const styleClass = {
    headerStyle: `text-primary-100  sm:text-xl md:text-2xl lg:text-3xl text-center font-semibold `,
    paragrphStyle: `  sm:text-sm md:text-base lg:text-lg text-center `,
  };

  return (
    <div className="flex">
      <div className="  sm:w-full  md:w-full lg:w-6/12 p-10  ">
        <h1 className={styleClass.headerStyle } >{t('dontDonate.header')} </h1>
        <p className={styleClass.paragrphStyle} >
          <br /> {t('dontDonate.firstParagraph')}<br />
          <br /> {t('dontDonate.secondParagraph')}<br />
          <br />
          {t('dontDonate.thirdParagraph')}
        </p>
      </div>

      <div className="  sm: hidden  md: hidden 	lg:flex w-6/12  p-10 ">
        <img src={image} alt="donateImage " />
      </div>
    </div>
  );
}

export default DontDonate;
