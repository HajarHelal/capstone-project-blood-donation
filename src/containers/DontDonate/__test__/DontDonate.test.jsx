import React from 'react';
import renderer from 'react-test-renderer';
import DontDonate from '../DontDonate';

// tests will use to compare to.
it('renders correctly', () => {
  const tree = renderer.create(<DontDonate />).toJSON();
  expect(tree).toMatchSnapshot();
});
