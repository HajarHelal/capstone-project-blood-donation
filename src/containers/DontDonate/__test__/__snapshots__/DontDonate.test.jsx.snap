// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex"
>
  <div
    className="  sm:w-full  md:w-full lg:w-6/12 p-10  "
  >
    <h1
      className="text-primary-100  sm:text-xl md:text-2xl lg:text-3xl text-center font-semibold "
    >
      dontDonate.header
       
    </h1>
    <p
      className="  sm:text-sm md:text-base lg:text-lg text-center "
    >
      <br />
       
      dontDonate.firstParagraph
      <br />
      <br />
       
      dontDonate.secondParagraph
      <br />
      <br />
      dontDonate.thirdParagraph
    </p>
  </div>
  <div
    className="  sm: hidden  md: hidden 	lg:flex w-6/12  p-10 "
  >
    <img
      alt="donateImage "
      src="doctor-office-and-blood-donation-design-vector-22887400.jpg"
    />
  </div>
</div>
`;
